### DISCUSSION TITLE
米游社

### DISCUSSION THREAD

**Discussion by Yingl-1** [AIsouler/GKD_subscription#140] [Q&A]
米游社的星铁和原神签到自动退出出错无法退出，会进入这个游戏的主页

**Comment by AIsouler** [comment]
快照发一下

**Reply by Yingl-1** [reply]
https://i.gkd.li/i/14964096
//签到页面（星铁）

https://i.gkd.li/i/14964089
//签到后出的提示（星铁）

https://i.gkd.li/i/14964069
//签到后出进入的（星铁）

https://i.gkd.li/i/14964136
//未触发。今天（原神）

**Reply by AIsouler** [reply]
你的快照没有截图，授权一下截图权限
![image](https://github.com/AIsouler/GKD_subscription/assets/57941037/1b1d6258-55ea-45d8-8824-09d982437ce1)

**Reply by Yingl-1** [reply]
https://i.gkd.li/i/14967627
//原神没反应
https://i.gkd.li/i/14967631
https://i.gkd.li/i/14967633
//星铁反应了，不过这次没进主页，只不过看第二个链接一直在gkd，所以按理说是要进主页的，我的是，但是不知道为什么没进，一直在gkd
PS。这是我借的号弄得

**Comment by AIsouler** [comment]
已修复 dd25a5399ad6b98f369b2f961c89af5b9a50643a

**Reply by Yingl-1** [reply]
https://i.gkd.li/i/14981609
//原神签到后不反回

**Reply by AIsouler** [reply]
9542a9d467fd40ba2bb21f73fc316866b61eb46d

**Reply by Yingl-1** [reply]
希望可以加个幕布的
https://i.gkd.li/i/15066979

**Reply by AIsouler** [reply]
bfd324ba031e19a9aea2a0650e52ccd3bc4d64e7

**Reply by Yingl-1** [reply]
https://i.gkd.li/i/15223416
番茄阅读
